content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -172,9 +179,7 @@ export class _CollapsibleSection extends React.PureComponent {
               <span className="learn-more-link-wrapper">
                 {learnMore &&
                   <span className="learn-more-link">
-                    <a href={learnMore.link.href}>
-                      <FormattedMessage id={learnMore.link.id} />
-                    </a>
+                    <a href={learnMore.link.href} data-l10n-id={learnMore.link.id}>{learnMore.link.text}</a>
```

Comment:
I see we have a `getFormattedMessage(message)` helper here as well as in at least Sections.jsx. Seems like we might want to refactor this to take a `message: string | {id: …, values: …}` and perhaps make it a higher order component to allow setting the data-l10n-id attribute on the desired element vs just setting plain text child? This could be a followup